symphony-bdk-http/symphony-bdk-http-webclient/src/main/java/com/symphony/bdk/http/webclient/ApiClientBuilderWebClient.java

```diff
@@ -0,0 +1,165 @@
+package com.symphony.bdk.http.webclient;
+
+import com.symphony.bdk.http.api.ApiClient;
+import com.symphony.bdk.http.api.ApiClientBuilder;
+import com.symphony.bdk.http.api.util.ApiUtils;
+
+import io.netty.channel.ChannelOption;
+import io.netty.handler.ssl.SslContext;
+import io.netty.handler.ssl.SslContextBuilder;
+import io.netty.handler.timeout.ReadTimeoutHandler;
+import org.springframework.http.client.reactive.ReactorClientHttpConnector;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.netty.http.client.HttpClient;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.TrustManagerFactory;
+
+public class ApiClientBuilderWebClient implements ApiClientBuilder {
+
+  private final Map<String, String> defaultHeaders;
+  private String basePath;
+  private byte[] keyStoreBytes;
+  private String keyStorePassword;
+  private byte[] trustStoreBytes;
+  private String trustStorePassword;
+  private int connectionTimeout;
+  private int readTimeout;
+  private String temporaryFolderPath;
+
+  public ApiClientBuilderWebClient() {
+    basePath = "";
+    keyStoreBytes = null;
+    keyStorePassword = null;
+    trustStoreBytes = null;
+    trustStorePassword = null;
+    defaultHeaders = new HashMap<>();
+    connectionTimeout = 15000;
+    readTimeout = 60000;
```

Comment:
perhaps we should share default timeouts for all our implementations at the API level?